src/agents/synthetic_worker.py

```
@@ -0,0 +1,74 @@
+
+        return {
+            "success": True,
+            "agent": self.name,
+            "task": task,
+            "call_count": self._calls,
+            "correlation_id": context.metadata.get("correlation_id"),
+            **payload,
+        }
+
+    @staticmethod
+    def _to_positive_int(value: Any, *, default: int) -> int:
```

Ruff (0.15.5)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `value`

(ANN401)